src/utils/explore.service.js

```diff
@@ -0,0 +1,16 @@
+import { EnvironmentUrl } from "./environment/environment-url.model";
+import { getCurrentEnvironment } from "./environment/environment.service";
+
+const PROD_ENV = ["ANVIL-PROD", "MASTER", "PROD"];
```

Comment:
@NoopDog the PROD_ENV array of potential PROD environments is an attempt to catch all possible PROD env that require the explore URL to point to PROD....